easybuild/easyconfigs/b/Blender/Blender-2.66a-foss-2016.04-Python-3.3.2.eb

```diff
@@ -21,17 +21,12 @@ configopts += '-DWITH_GAMEENGINE=OFF '
 configopts += '-DWITH_CYCLES=OFF '
 configopts += '-DWITH_SYSTEM_GLEW=OFF '
 
-# Python paths
-configopts += '-DPYTHON_LIBRARY=${EBROOTPYTHON}/lib '
-configopts += '-DPYTHON_INCLUDE_DIR=${EBROOTPYTHON}/include/python3.5m '
```

Comment:
@boegel I didn't need them for Python 3.3.2 when I built it for some reason but I'll include them back in.